Documentation/installation/k8s-install-helm.rst

```diff
@@ -85,21 +85,19 @@ Install Cilium
 
        .. include:: ../installation/requirements-aks.rst
 
+       .. note::
+            Installing Cilium via helm is supported only for AKS BYOCNI cluster and
+            not for Azure CNI Powered by Cilium clusters.
+
        **Install Cilium:**
 
        Deploy Cilium release via Helm:
 
        .. parsed-literal::
-
```

Comment:
No need to remove this line, for my part I prefer keeping a blank line to separate the directive name from its content.